scikeras/wrappers.py

```diff
@@ -676,11 +638,11 @@ def _filter_params(self, fn, params_to_check=None):
 
     def _get_param_names(self):
         """Get parameter names for the estimator"""
-        parameters = super()._get_param_names()
-        # add kwargs/sk_params if the user gave those as input
-        if self._sk_params:
-            return sorted(parameters + self._sk_params)
-        return parameters
+        return (
+            k
+            for k in self.__dict__
+            if not k.endswith("_") and not k.startswith("_")
+        )
```

Comment:
This is borrowed from SciKeras. For where we are headed, I think this makes more sense than storing `_sk_params`. It'll be up to users to make sure they don't set any params without `_` that they don't want to show up in `get_params`.

Reply:
I added `k.startswith("_")` to accomodate parameters like `self._random_state`. So the rule will be that all model paramers (i.e. the `__init__` ones) cannot start/end with `_`, at least for now. We'll have to see how that plays in with my overwriting proposal in https://github.com/adriangb/scikeras/pull/47#discussion_r469709646